models/balancer/optimism/balancer_v2_optimism_transfers_bpt.sql

```diff
@@ -0,0 +1,39 @@
+{{
+    config(
+        alias='transfers_bpt',
+        post_hook='{{ expose_spells(\'["optimism"]\',
+                                    "project",
+                                    "balancer_v2",
+                                    \'["stefenon"]\') }}'
+    ) 
+}}
+
+{% set event_signature = '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef' %}
+{% set project_start_date = '2022-05-19' %}
+
+WITH registered_pools AS (
+    SELECT
+      DISTINCT poolAddress AS pool_address
+    FROM
+      {{ source('balancer_v2_optimism', 'Vault_evt_PoolRegistered') }}
+  )
+
+SELECT DISTINCT * FROM (
+    SELECT
+        logs.contract_address,
+        logs.tx_hash AS evt_tx_hash,
+        logs.index AS evt_index,
+        logs.block_time AS evt_block_time,
+        logs.block_number AS evt_block_number,
```

Comment:
You can add a `block_date` column to optimize incremental partition
```suggestion
        logs.index AS evt_index,
        logs.block_time AS evt_block_time,
        TRY_CAST(date_trunc('DAY', logs.block_time) AS date) AS block_date,
        logs.block_number AS evt_block_number,
```